libs/scala.json

```diff
@@ -1,7 +1,11 @@
 [
-  {"id": "scala.cats", "imports": ["cats."], "name": "Cats", "repo": "typelevel/cats", "tags": ["monads"], "tech": "functional"},
-  {"id": "scala.spire", "imports": ["spire."], "name": "Spire", "repo": "non/spire", "tags": ["numbers", "math"], "tech": "utils"},
-  {"id": "scala.shapeless", "imports": ["shapeless."], "name": "Shapeless", "repo": "milessabin/shapeless", "tags": ["types"], "tech": "utils"},
-  {"id": "scala.circe", "imports": ["io.circe."], "name": "Circe", "repo": "circe/circe", "tags": ["json", "json-transformation"], "tech": "app-data"},
-  {"id": "scala.scalatest", "imports": ["org.scalatest."], "name": "ScalaTest", "repo": "scalatest/scalatest", "tags": ["unit-testing"], "tech": "testing"}
+  {"id": "scala.cats", "imports": ["cats."], "name": "Cats", "repo": "typelevel/cats", "tags": ["monads"], "tech": ["functional"]},
+  {"id": "scala.spire", "imports": ["spire."], "name": "Spire", "repo": "non/spire", "tags": ["numbers", "math"], "tech": ["utils"]},
+  {"id": "scala.shapeless", "imports": ["shapeless."], "name": "Shapeless", "repo": "milessabin/shapeless", "tags": ["types"], "tech": ["utils"]},
+  {"id": "scala.circe", "imports": ["io.circe."], "name": "Circe", "repo": "circe/circe", "tags": ["json", "json-transformation"], "tech": ["app-data"]},
+  {"id": "scala.scalatest", "imports": ["org.scalatest."], "name": "ScalaTest", "repo": "scalatest/scalatest", "tags": ["unit-testing"], "tech": ["testing"]},
+  {"id": "scala.play", "imports": ["play."], "name": "Play Framework", "repo": "playframework/playframework", "tags": ["applications"], "tech": ["scala-web"]},
```

Comment:
We usually assign the tag 'framework' for web frameworks. But if you think 'applications' is more appropriate, we can use that.

Reply:
`framework` makes more sense. Updated